e2e/driver/ChromeDriver.ts

```diff
@@ -52,11 +51,10 @@ export class ChromeDriver implements IDriver {
             .forBrowser('chrome')
             .setChromeOptions(options);
 
-        // if 'true' run with remote driver
+            // if 'true' run with remote driver
         if (TestConstants.TS_SELENIUM_REMOTE_DRIVER_URL) {
             builder = builder.usingServer(TestConstants.TS_SELENIUM_REMOTE_DRIVER_URL);
         }
```

Comment:
It looked more readable with empty line.

Reply:
Fixed

Reply:
Actually, it wasn't fixed.

Reply:
I meant empty line between ` return builder;`.